.eslintignore

```diff
@@ -11,3 +11,7 @@
 /website/build/
 /website/static/playground.js
 /website/static/lib/
+
+# This is temporal, will be removed when ESLint supports Top-Level await.
+# https://github.com/eslint/eslint/issues/14632
+/scripts/generate-changelog.mjs
```

Comment:
How about not use top-level await for now? We can use [`unicorn/prefer-top-level-await`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-top-level-await.md) when ESLint is ready.

Reply:
Sounds good. It is a good ESLint rule.